torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1530,7 +1545,7 @@ def _verify(expr, potential_expr):
             for i, ss in enumerate(t.size()):
                 property_source = TensorPropertySource(source, TensorProperty.SIZE, i)
                 track_symint(property_source, ss)
-                if _is_dim_dynamic(t, i):
+                if dyn_dims and i in dyn_dims:
```

Comment:
this is annoying, can you force the user to always pass in a set (could be empty)